tailwind.config.ts

```diff
@@ -1,6 +1,7 @@
 import type { Config } from 'tailwindcss';
 import { formatRgb } from 'culori';
 import plugin from 'tailwindcss/plugin';
+import 'dotenv/config'; // TODO: we can remove this with node@20.6.0
```

Comment:
Hmm are we not using node@20 already ?

Reply:
Copied this from playwright and left the comment as it felt out of scope to address removing dotenv. See this PR: https://github.com/bancorprotocol/carbon-app/pull/1543